spec/forms/provider_interface/offer_wizard_spec.rb

```diff
@@ -57,6 +70,59 @@
     end
   end
 
+  describe '.build_from_application_choice' do
+    let(:application_choice) do
+      create(
+        :application_choice,
+        course_option: create(:course_option),
+        offer: offer,
+      )
```

Comment:
```suggestion
      create(
        :application_choice,
        offer: offer,
      )
```
The default application_choice factory creats a course_option, so I think that line can be removed.
https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/spec/factories/application_choice.rb#L3